include/dsn/c/api_layer1.h

```diff
@@ -689,23 +691,25 @@ typedef union dsn_global_partition_id
     uint64_t value;
 } dsn_gpid;
 
-inline uint64_t dsn_gpid_to_hash(dsn_gpid gpid)
+inline int32_t dsn_gpid_to_hash(dsn_gpid gpid)
```

Comment:
will fix in latter pr

On Wed, Aug 3, 2016 at 4:20 PM, Zhenyu Guo <notifications@github.com> wrote:

> In include/dsn/c/api_layer1.h
> <https://github.com/imzhenyu/rDSN/pull/514#discussion_r73297983>:
>
> > @@ -689,23 +691,25 @@ typedef union dsn_global_partition_id
> >      uint64_t value;
> >  } dsn_gpid;
> >
> > -inline uint64_t dsn_gpid_to_hash(dsn_gpid gpid)
> > +inline int32_t dsn_gpid_to_hash(dsn_gpid gpid)
>
> dsn_gpid_to_thread_hash?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/imzhenyu/rDSN/pull/514/files/277556d0ee3cfbf77687db90267a8d77ab1df29f#r73297983>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAgQPSd8wWgKuEmGP1lqJoOcrWklNKDnks5qcE9HgaJpZM4JbaGd>
> .
>